AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.java

```diff
@@ -3688,5 +3729,34 @@ public boolean ankiIsActiveNetworkMetered() {
                 return true;
             }
         }
+
+        @JavascriptInterface
+        public void ankiJsConsoleLog(boolean enableLog) {
+            if (enableLog) {
+                enableDebugMode = true;
+
+                View parentLayout = findViewById(android.R.id.content);
+
+                Snackbar snackbar = Snackbar.make(parentLayout, "Bebug Mode Enabled", Snackbar.LENGTH_LONG);
```

Comment:
Also should be extracted to a string resource (`02-strings.xml` should do)
```suggestion
                Snackbar snackbar = Snackbar.make(parentLayout, "Debug Mode Enabled", Snackbar.LENGTH_LONG);
```